Libraries/CMSIS/Device/Maxim/MAX32672/Include/aeskeys_regs.h

```diff
@@ -84,7 +86,11 @@ extern "C" {
 /**
  * @ingroup aeskeys_registers
```

Comment:
I wonder if we can also clean up our Doxygen output for deprecated functions/structs.

Primarily we can add [@deprecated](https://www.doxygen.nl/manual/commands.html#cmddeprecated) to the header for the structs and register modules.

I'm also not sure exactly how the [current Doxygen deprecated list](https://analog-devices-msdk.github.io/msdk/Libraries/PeriphDrivers/Documentation/MAX32672/deprecated.html) is being generated.  There's not a `@deprecated` tag anywhere in the file.

Reply:
Ope, I didn't know @deprecated was actually used by Doxygen. I originally added the annotation into our deprecated register files, so the register workflows can filter out deprecated register files.